hwdb.d/60-keyboard.hwdb

```diff
@@ -1567,7 +1567,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnMicro-Star*:pn*Modern*:*
 
 evdev:name:MSI Laptop hotkeys:dmi:bvn*:bvr*:bd*:svn*:pnM[iI][cC][rR][oO]-S[tT][aA][rR]*:*
  KEYBOARD_KEY_0213=touchpad_on
- KEYBOARD_KEY_0214=f23
+ KEYBOARD_KEY_0214=touchpad_off
```

Comment:
Sending touchpad_on/_off  for scancodes 0213 / 0214 has been the default for the kernel since support for these (fake) scancodes was added. So this entry can be dropped.